packages/parser/tests/tools/ts-error-serializer.ts

```diff
@@ -0,0 +1,18 @@
+import type { Plugin } from 'pretty-format';
+import { TSError } from '@typescript-eslint/typescript-estree/dist/node-utils';
+
+export const serializer: Plugin = {
```

Comment:
this is copy of serializer from `typescript-estree`, we can't just inport it as sertializers are not compiled/exposed